configs/celeryd

```diff
@@ -23,8 +23,9 @@ CELERYD_PID_FILE="/var/run/celery/%n.pid"
 # Workers should run as an unprivileged user.
 #   You need to create this user manually (or you can choose
 #   a user/group combination that already exists (e.g., nobody).
-CELERYD_USER="celery"
-CELERYD_GROUP="celery"
+CELERYD_USER=`id -un`
```

Comment:
Can you add a comment to explain why we don't use root? From offline chat it's because root can't access the code directory(so who will access? an IDE created user?)

I'm also little bit surprised we can use bash script here, amazing!

Reply:
@dichenqiandbt Good callout, i added a comment. You are correct, the IDE creates a user `999` and access to the project files is restricted to that user.

This was originally a shot in the dark and I was surprised this worked as well, but these variables are just used in `celeryd.sh` as part of a command, so it seems fine so far

Reply:
Yeah the IDE runs as a specific user (999) and all files/directories in the container are locked down to that user as well